Dockerfile-testing

```diff
@@ -0,0 +1,34 @@
+# Use the official Python base image
+FROM python:3.11-bookworm
+
+# Set the working directory in the container
+WORKDIR /app
+
+# Copy the requirements file to the container
+COPY requirements.txt .
+
+# Install build-essential
+RUN apt-get update && apt-get install -y build-essential
+
+# Clean up
+RUN apt-get clean && rm -rf /var/lib/apt/lists/*
+
+ADD https://github.com/bitwarden/sdk/releases/download/bws-v0.4.0/bws-x86_64-unknown-linux-gnu-0.4.0.zip /tmp
```

Comment:
Is this the official installation method for the bitwarden SDK?

Reply:
Yes https://bitwarden.com/help/secrets-manager-cli/

I didn't see they had an sdk before might switch to that https://bitwarden.com/help/secrets-manager-sdk/

Reply:
#96 #95